scala-generator/src/main/scala/models/Play2Json.scala

```diff
@@ -77,9 +77,19 @@ case class Play2Json(
   }
 
   private[models] def enumReadersAndWriters(enum: ScalaEnum): String = {
-    val jsObjectWriterMethod = play2JsonCommon.toJsonObjectMethodName(ssd.namespaces, enum.name)
     val jsValueWriterMethod = play2JsonCommon.implicitWriterName(enum.name)
-    val implicitWriter = play2JsonCommon.implicitWriter(enum.name, enum.qualifiedName, jsValueWriterMethod)
+    val jsObjectWriterMethod = play2JsonCommon.toJsonObjectMethodName(ssd.namespaces, enum.name)
```

Comment:
Should we modify `jsObjectWriterMethod` to be fully qualified like `jsValueWriterMethod` is?

```
 jsValueWriterMethod => "jsonWritesApidocExampleUnionTypesDiscriminatorSystemUser"
jsObjectWriterMethod => "jsObjectSystemUser"
```

Reply:
I did not update that logic. I don't think the fqcn brings value though, given the name itself is unique and sufficient in this context (and scala package).

@benwaffle Ok to leave as is?

Reply:
fine to leave